chore(store): adding a Docker compose option that allows to compare postgres & sqlite metrics

Ivansete-status opened the pull request:
## Summary
Adding a docker compose file so SQLite & Postgres performance can be compared quickly and simultaneously.

## Details

This aims to create a base environment where we can analyze performance and tweak the database, etc.

The docker compose file creates the next containers:

1. nwaku with _Store_ (Postgres.)
2. nwaku with _Store_ (SQLite.)

( 1. & 2. are interconnected )

3. msg_publisher. This contains an infinite loop that sends `json rpc` requests to 1. and 2. so that both publish messages.
4. store_query_producer. This is similar to the 3. but in this case, it performs _Store_ requests to retrieve historical messages from 1. and 2.
5. grafana; prometheus; postgres-exporter (notice that for now, I'm not adding _Postgres_ dashboard. To be added in the future)

The _compose_ is built in a way that the data stored by both databases will outlive the containers' lifetimes. 

In the _Grafana_ dashboard_ we get information like this:

![Screenshot from 2023-08-24 13-23-42](https://github.com/waku-org/test-waku-query/assets/128452529/f38b1923-3139-4268-bbdf-e45ac8728582)

note: I'm deleting _postgres-docker-compose.yml_ as it becomes redundant after this change is applied.

## Issue
closes https://github.com/waku-org/nwaku/issues/1894

Ivansete-status committed:
adding a Docker compose option that allows to compare postgres & sqlite metrics

vpavlin commented on docker/store_query_producer.sh:
```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+## Install the `dig` command
+apk --no-cache add bind-tools
```
The file name is strange - what is this script "producing"? Is it meant as "producing the store query message"?

Second question - how long does it take to start this one and the publisher? It feels weird to have to install the deps everytime you want to (re)start the test. Maybe add a dockerfile with the packages installation and use an image built from that?

Ivansete-status replied:
Thanks for the comments!

The `store_query_producer.sh` just produces _Store_ requests. So yes, is aimed to generate "store query messages" so that we have traffic from the message retrieval.

https://github.com/waku-org/nwaku/blob/4539dfc76109382f321080fdfbf92e44531c6de8/waku/node/waku_node.nim#L610

That makes me think that I'm missing a configuration setting for the nodes, which is the `storenode` param.

The main purpose of this script is to force the `nwaku` node, to perform a query request to the other node.

I'm open to renaming it to a more appropriate name.

Re the second question, these scripts are fast to start, even having those installations. The thing is that I hard-coded a 20s wait to make sure the `nwaku` nodes had started completely.

Ivansete-status committed:
Setting storenode counter parts

With this parameter, each node is configured as "store client" and the
"store server" is the one indicated in the "storenode" parameter.

Ivansete-status committed:
Grafana dashboard: setting the default period time to "last 5 min"

jm-clius reviewed (approved):
LGTM. Useful tool. As mentioned elsewhere, it will be good to publish some basic results of these comparative tests in a write-up somewhere.

Ivansete-status commented:
> LGTM. Useful tool. As mentioned elsewhere, it will be good to publish some basic results of these comparative tests in a write-up somewhere.

Thanks for the comment @jm-clius ! We will have the _Postgres_ related info in https://www.notion.so/Postgres-e33d8e64fa204c4b9dcb1514baf9c582
